Directory.Build.props

```diff
@@ -0,0 +1,78 @@
+<Project>
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+	<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+	<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <TargetFramework Condition="'$(Mono)' == ''">net5.0</TargetFramework>
+    <TargetFramework Condition="'$(Mono)' != ''">netstandard2.1</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Optimize>true</Optimize>
+    <UseVSHostingProcess>false</UseVSHostingProcess>
+    <LangVersion>7.3</LangVersion>
+    <DebugSymbols>true</DebugSymbols>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AutoGenerateBindingRedirects>false</AutoGenerateBindingRedirects>
+    <RootNamespace>OpenRA</RootNamespace>
```

Comment:
This causes some problems with the namespaces for all the projects that is not named `OpenRA`

Reply:
I watched how to build ra2 and thought all mods like this. But OpenSA use root directory of engine. Yeh, ```<RootNamespace>OpenRA</RootNamespace>``` need remove. (revert to OpenRA.Game.csproj ?)

Reply:
Please remove this line